### DISCUSSION TITLE
Some orders showing only Parent SKU’s – not variation skus

### DISCUSSION THREAD

**Discussion by ghost** [woocommerce/woocommerce#36373] [General]
I have this really annoying issue that arose very recently. Within the last few days, and it is getting worse.

Some orders placed by customers do not show the variation SKU the customer selected; only the parent SKU.

For example, I will see Serum (parent) instead of Serum-M (variant) on a customer order, leaving me to guess what they want.

It seems like something that would be caused by adding items to cart without selecting the variant characteristics first (size, color), but I have tried to do this in troubleshooting. My site will not allow me to add an item to cart without having a size and color selected, which is how it should be.

So how are some peoples orders coming through as Parent SKUs instead of the proper variants? It’s driving me crazy, and it seems completely random. Some orders have the variants appear as they should. Other orders only show the parent SKU. No apparent rhyme or reason.

This is becoming a serious issue because I can’t just guess what the customer wants, and in some cases, 

I’ve searched on this issue and there are no blatant solutions. Can anyone point me in the right direction? Thank you.

**Comment by mojzesz777** [comment]
"My site will not allow me to add an item to cart without having a size and color selected, which is how it should be."
From what you write, you would have to have a simple variant and use additional product fields that you could indicate whether they are required or not. See if it would solve your need: https://www.wpdesk.pl/sklep/flexible-product-fields-pro-woocommerce/
